packages/clerk-js/src/ui/components/SignIn/SignIn.tsx

```diff
@@ -145,14 +147,16 @@ function SignInRoutes(): JSX.Element {
 
 function SignInRoot() {
   const signInContext = useSignInContext();
+  const normalizedProps = {
+    componentName: 'SignUp',
+    ...signInContext.__experimental?.combinedProps,
+    emailLinkRedirectUrl: signInContext.emailLinkRedirectUrl,
+    ssoCallbackUrl: signInContext.ssoCallbackUrl,
+    ...normalizeRoutingOptions({ routing: signInContext?.routing, path: signInContext?.path }),
+  } as SignUpContextType;
```

Comment:
Does this work?

```suggestion
  } satisfies SignUpContextType;
```

Reply:
It does not, I remember @dstaley mentioning this was ok for now though.